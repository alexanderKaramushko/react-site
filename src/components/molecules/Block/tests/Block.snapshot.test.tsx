import React from 'react';
import renderer from 'react-test-renderer';
import { Block } from '..';

describe('>>>B L O C K --- Snapshot', () => {
    it('+++capturing Snapshot of Block', () => {
        const renderedValue = renderer.create(<Block title="Test" />).toJSON();
        expect(renderedValue).toMatchSnapshot();
    });
});
