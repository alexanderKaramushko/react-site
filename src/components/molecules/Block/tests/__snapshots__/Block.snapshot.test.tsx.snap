// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`>>>B L O C K --- Snapshot +++capturing Snapshot of Block 1`] = `
<div
  className=""
>
  <h2>
    Test
  </h2>
  
  0
</div>
`;
